iOS/Flows/Conversation/Cells/TimeMachineCollectionViewLayout.swift

```diff
@@ -481,17 +496,26 @@ class TimeMachineCollectionViewLayout: UICollectionViewLayout {
 
     override func finalLayoutAttributesForDisappearingItem(at itemIndexPath: IndexPath)
     -> UICollectionViewLayoutAttributes? {
+        guard deletedIndexPaths.contains(itemIndexPath) else { return nil }
+
+        return super.finalLayoutAttributesForDisappearingItem(at: itemIndexPath)
```

Comment:
Ok so here's the less hacky version. If this item isn't actually being deleted, return nil for the final layout attributes. This essentially means don't do any animations, just go directly to where you're supposed to be.